app.py

```diff
@@ -187,6 +187,7 @@ def home_content():
 
 pg = st.navigation(
     {
+        "Navigation": [home, docs, changelog],
```

Comment:
it's more intuitive to me to have the navigation tabs above the list of evaluation categories

Reply:
This is how it was originally but a person suggested to change the order 😄

Reply:
Hahah, there's no pleasing everyone! I don't feel strongly at all so will switch it back